guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>Once the explanatory variables or the explanatory rasters list is ready, along with the <code>preprocessors</code> transformation,  these <s>are</s> <u>can</u> now <u>be </u>used as input for the <code>prepare_tabulardata()</code> method. The method takes the <code>feature layer</code> or a <code>spatial dataframe</code> containing the dataset and prepares it for fitting the model as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>